model/logs/mobile-crashes.yaml

```diff
@@ -0,0 +1,59 @@
+groups:
+  - id: mobile-crash-event
+    type: event
+    brief: >
+      This document defines attributes for crashes represented using Log Record Events.
+      `event.domain` is 'device' and `event.name` is 'crash
+    attributes:
+      - id: os.name
```

Comment:
@tonzhan2 When you are reusing existing attributes, you need to reference them. (Here is an example on how to do that: https://github.com/open-telemetry/semantic-conventions/blob/39af37d0935f7e9030cddade5b0e16a9c4ea135c/model/trace/http.yaml#L10).

In particular, when you are specifying `id` and `type` you are defining an attribute, but those attributes already exist. So, that would cause some conflicts.